pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -383,6 +406,20 @@ func scTemplateToRPC(cfg binding.Config, ctr ContractTmpl, imports map[string]st
 			ctr.SafeMethods[i].CallFlag = "Array"
 		case "*stackitem.Map":
 			ctr.SafeMethods[i].CallFlag = "Map"
+		case "[]bool":
+			ctr.SafeMethods[i].CallFlag = "ArrayOfBools"
+		case "[]*big.Int":
+			ctr.SafeMethods[i].CallFlag = "ArrayOfBigInts"
+		case "[][]byte":
+			ctr.SafeMethods[i].CallFlag = "ArrayOfBytes"
```

Comment:
We can't properly distinguish array of signatures and array of bytes here, then maybe we need to add a separate type for them.

Reply:
That's true because our crypto library always uses `[]byte` for signatures, both in and out.